### PR TITLE
HPCC-25846 copy configmgr/* and config2mgr/* to /etc/HPCCSystems/

### PR DESCRIPTION
**Pull request by xwang2713**
Copy <INSTALL_DIR>/etc/HPCCSystems/[configmgr|config2mgr] to /etc/HPCCSystems/

Before configuration refactoring $CCONFIG_DIR means /etc/HPCCSystems/ instead INSTALL_DIR>/etc/HPCCSystems/ for some executables.  Copying <INSTALL_DIR>/etc/HPCCSystems/[configmgr|config2mgr] to /etc/HPCCSystems/  makes these executables compatible with configuration refactoring.

Signed off by Xiaoming Wang <xiaoming.wang@lexisnexis.com>

 
## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [x] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [x] The commit message title makes sense in a changelog, by itself.
  - [x] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [ ] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [ ] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Cloud-compatibility
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [ ] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Smoketest:
- [ ] Send notifications about my Pull Request position in Smoketest queue.
- [ ] Test my draft Pull Request.

## Testing:

Tested with evngen

### CODE REVIEW

**Comment by xwang2713** [comment]
I update to a hard-copy instead of symbolic link. So there will be original files and avoid uninstall clean-up

File: initfiles/bash/etc/init.d/install-init.in [hunk 1]
```diff
@@ -197,6 +197,12 @@ installFile "$binPath/eclplus" "/usr/bin/eclplus" 1 || exit 1
 installFile "$binPath/wuget" "/usr/bin/wuget" 1 || exit 1
 installFile "$configs/$environment" "$sourcedir/$environment" 0 "$sourcedir"
 
+installFile "$path${CONFIG_DIR}/configmgr/configmgr.conf"  \
+     "${CONFIG_DIR}/configmgr/" 0 "${CONFIG_DIR}/configmgr" || exit 1
+installFile "$path${CONFIG_DIR}/configmgr/esp.xml" "${CONFIG_DIR}/configmgr/" 0 || exit 1
+installFile "$path${CONFIG_DIR}/configmgr/config2mgr.conf"  \
```
Comment (Michael-Gardner) [inline initfiles/bash/etc/init.d/install-init.in:203]: After testing, I don't see this file laid down.  It generates the /etc/config2mgr directory and remains empty. Looking through the source I wasn't finding a reference to a config2mgr.conf file anywhere.  Has that been implemented yet?

**Review by Michael-Gardner — APPROVED** [review]
Works fine, just would like some clarification on the config2mgr.conf file not existing. If this is the expected behavior then we're fine.

File: initfiles/bash/etc/init.d/install-init.in (hunk 1 above)
Comment (xwang2713) [reply initfiles/bash/etc/init.d/install-init.in]: Remove config2mgr.conf and add esp.xml

**Comment by xwang2713** [comment]
@richardkchapman this is ready to merge.

**Comment by richardkchapman** [comment]
@Michael-Gardner Have you approved this change?

**Review by Michael-Gardner — APPROVED** [review]
looks good

**Pull request merged**